src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            int buy = 0;
+            int supply = 0;
+            Path toPath = new File(toFileName).toPath();
+            String line = reader.readLine();
+            while (line != null) {
+                switch (line.split(",")[OPERATION]) {
```

Comment:
Create a String array variable inside the loop and reuse it instead of doing a split every single time 
```suggestion
                String[] lines = line.split(",");
                switch (lines[OPERATION]) {
```